chainer/link_hooks/spectral_normalization.py

```diff
@@ -108,7 +108,9 @@ class SpectralNormalization(link_hook.LinkHook):
         n_power_iteration (int): Number of power iteration.
             The default value is 1.
         eps (float): Numerical stability in norm calculation.
-            The default value is 1e-12.
+            The default value is 1e-6 for the compatibility with
```

Comment:
I changed eps to 1e-6 and mentioned the author's implementation use a different value.
https://github.com/chainer/chainer/pull/6655#discussion_r273389355